### PR TITLE
OTS-1200 Review accelerator-annotation-ios

### PR DESCRIPTION
**Pull request by PanoramicRum**
* Add Contributing Guidelines.
     * Add version and license badge to README.
     * Review README.
     * Add CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# OpenTok Annotations Accelerator for iOS CHANGELOG
+All notable changes to this project will be documented in this file.
+
+--------------------------------------
+
+####[2.0.0-beta7]
```
Comment (Lucashuang0802) [inline CHANGELOG.md]: This starts from 1.0.0-beta1. 😃

File: README.md [hunk 2]
```diff
@@ -1,24 +1,52 @@
-![logo](../tokbox-logo.png)
+![logo](./tokbox-logo.png)
 
-# OpenTok Annotations Accelerator Pack for iOS<br/>Version 1.0-Beta
+# OpenTok Annotations Accelerator Pack for iOS<br/>
```
Comment (Lucashuang0802) [inline README.md]: This starts from 1.0.0-Beta1 😃

File: README.md [hunk 3]
```diff
@@ -110,9 +140,18 @@ self.annotator.dataSendingHandler = ^(NSArray *data, NSError *error) {
...
-2. Your device must be running iOS 8 or later.
+
+## License
+
+This project is under the [MIT License](./LICENSE)
+
+
+# About OpenTok
+
+![logo](./tokbox-logo.png)
```
Comment (marinaserranomontes) [inline README.md]: @PanoramicRum IMHO, we should not duplicate logo in the same Readme. Also main logo is not visible (could you review the img route?)

Comment (PanoramicRum) [inline README.md]: I agree. I removed to logo from the top of the README. I can add it again if we decide so.

File: CHANGELOG.md (hunk 1 above)
Comment (PanoramicRum) [inline CHANGELOG.md]: Thanks, fixed it :+1:

File: README.md (hunk 2 above)
Comment (PanoramicRum) [inline README.md]: Fixed it too.

**Pull request merged**